github/util.py

```diff
@@ -308,6 +316,14 @@ def release_versions(self):
                 yield semver.parse_version_info(release.tag_name)
             except ValueError: pass # ignore
 
+    def release_tags(self):
+        return _.map(self.repository.releases(), 'tag_name')
```

Comment:
vanilla python3 alternatives:
return (release.tag_name for release in self.repository.releases())
return map(lambda release: release.tag_name, self.repository.releases())